From 0c6bea60acde4ec838a68c3f5b5a8ac02226a496 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 19 Aug 2014 19:30:57 -0400
Subject: [scsi] qla4xxx: Improve loopback failure messages

Message-id: <1408476658-5056-20-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88350
O-Subject: [PATCH RHEL 7.1 e-stor 19/20] qla4xxx: Improve loopback failure messages
Bugzilla: 1089349
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Nilesh Javali <nilesh.javali@qlogic.com>

scsi commit id 56ccb988ce242eb2272fae5ac38e28bbd08268cf

Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Reviewed-by: Mike Christie <michaelc@cs.wisc.edu>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_bsg.c b/drivers/scsi/qla4xxx/ql4_bsg.c
index 04a0027..9f92cbf 100644
--- a/drivers/scsi/qla4xxx/ql4_bsg.c
+++ b/drivers/scsi/qla4xxx/ql4_bsg.c
@@ -517,7 +517,7 @@ static int qla4_83xx_wait_for_loopback_config_comp(struct scsi_qla_host *ha,
       (ha->idc_extend_tmo * HZ))) {
     ha->notify_idc_comp = 0;
     ha->notify_link_up_comp = 0;
-    ql4_printk(KERN_WARNING, ha, "%s: IDC Complete notification not received",
+    ql4_printk(KERN_WARNING, ha, "%s: Aborting: IDC Complete notification not received",
         __func__);
     status = QLA_ERROR;
     goto exit_wait;
@@ -538,7 +538,7 @@ static int qla4_83xx_wait_for_loopback_config_comp(struct scsi_qla_host *ha,
   if (!wait_for_completion_timeout(&ha->link_up_comp,
        (IDC_COMP_TOV * HZ))) {
    ha->notify_link_up_comp = 0;
-   ql4_printk(KERN_WARNING, ha, "%s: LINK UP notification not received",
+   ql4_printk(KERN_WARNING, ha, "%s: Aborting: LINK UP notification not received",
        __func__);
    status = QLA_ERROR;
    goto exit_wait;
-- 
1.7.1